src/App.js

```diff
@@ -44,9 +45,15 @@ function logPageView(location, action) {
 	ReactGA.set({ page: location.pathname + location.search });
 	ReactGA.pageview(location.pathname + location.search);
 }
+ReactGA4.initialize(GA_TRACKING_ID);
```

Comment:
This needs to stay ReactGA.
We are just adding reactga4 so that we are ready to do the swap over. We are not ready to do that yet because we are revisiting what our events should look like. Revert this back to just use ReactGA, and we should be good to go.